docs/source/transducer.rst

```diff
@@ -0,0 +1,21 @@
+.. role:: hidden
+    :class: hidden-section
+
```

Comment:
I think it's a good idea to note that prototype is nightly-only feature at the top of all the pages for prototype. So that there is a less chance of receiving the report like [this](https://github.com/pytorch/text/issues/1079)

Reply:
I've labelled the issue as prototype and linked to the "source of truth" definition of prototype, which says "not part of release". Do you mean to say that you feel this may not be clear enough?

Reply:
Yes, I think we cannot be too careful here.

I suggest add `.. note::` or `.. warning::` and tell users to use nightly build in a more straightforward manner. 

Reading the classification, I feel like it is not even included in nightly build, which is another binary distribution.

> Prototype: These features are typically not available as part of binary distributions like PyPI or Conda, except sometimes behind run-time flags, and are at an early stage for feedback and testing.